lark/utils.py

```diff
@@ -113,8 +113,13 @@ def deserialize(cls, data, namespace, memo):
 except ImportError:
     regex = None
 
-import sre_parse
-import sre_constants
+try:  # Python 3.11
```

Comment:
I recommend using `sys.version_info` instead of `try` on `import`, it serves as documentation to the user, it enables the workaround to be dropped when that version of Python is dropped, and it can be used by static analysis tools like mypy and pyupgrade.

Reply:
You make a good point, thanks

Reply:
True, though older alpha / beta / rcs are not worth supporting after a new one comes out. Especially alphas once a beta is out. They are essentially just development snapshots.

Build 0.8.0 doesn’t support alpha 6 either.